charts/spire/values.yaml

```diff
@@ -1,8 +1,16 @@
-# You can enable features that affect all services here.
-# global:
-#   telemetry:
-#     prometheus:
-#       enabled: true
+# You can enable config/features that affect all services here.
+global:
+  spire:
+    # -- Set the name of the Kubernetes cluster
+    clusterName: "example-cluster"
+    # -- Set the trust domain to use for the spiffe identifiers
+    trustDomain: "example.org"
+    # -- Override all instances of bundleConfigMap
+    bundleConfigMap: ""
```

Comment:
Why don't we set the bundle here?

Reply:
https://github.com/spiffe/helm-charts/pull/156#discussion_r1155107271